utils/Merkelizer.js

```diff
@@ -49,15 +38,8 @@ export default class Merkelizer {
     );
   }
 
-  static codeHash (code) {
-    return ethers.utils.solidityKeccak256(
-      ['bytes'],
-      ['0x' + code.join('')]
-    );
-  }
-
   static stackHash (stack, sibling) {
-    let res = sibling || EMPTY_STATE;
+    let res = sibling || ZERO_HASH;
```

Comment:
I wrote this file https://github.com/leapdao/solEVM-enforcer/blob/master/test/helpers/hash.js
This supposes to do the same as your `stackHash` function.
But now I think about it, this file is better in `utils`, near this `Merkelizer`.
Could you create a `Hash.js` in `utils` that have all those hash functions for better reusability?

Reply:
@peara 
These are `static`, you can import the Merkelizer and use `Merkelizer.stackHash`.
I did that on purpose because the hashing algos belong to the Merkelizer.

Reply:
@peara I pushed a new commit. Merkelizer.stateHash mimics the Hash.slb implementation now 👍